src/applications/gi/containers/SearchPage.jsx

```diff
@@ -226,68 +226,102 @@ export class SearchPage extends React.Component {
+              />
+            </div>
+            <div className="row">
+              <div className="column">
+                <div className="vads-l-row vads-u-justify-content--space-between vads-u-align-items--flex-end">
+                  <div className="vads-l-col--10 site-grid-example">
+                    <h1>
+                      {!search.inProgress &&
+                        `${(count || 0).toLocaleString()} `}
+                      Search Results
+                    </h1>
+                  </div>
+                  <div className="vads-l-col--2 site-grid-example">
+                    <div className="vads-u-display--none small-screen:vads-u-display--block vettec-logo-container">
+                      <img
+                        className="vettec-logo"
+                        src={require('site/assets/img/logo/vet-tec-logo.png')}
+                        alt="Vet Tec Logo"
+                      />
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div>
+            <VetTecSearchForm
+              filtersClass={filtersClass}
+              search={search}
+              autocomplete={this.props.autocomplete}
+              location={this.props.location}
+              clearAutocompleteSuggestions={
+                this.props.clearAutocompleteSuggestions
+              }
+              fetchAutocompleteSuggestions={
+                this.props.fetchAutocompleteSuggestions
+              }
+              handleFilterChange={this.handleFilterChange}
+              updateAutocompleteSearchTerm={
+                this.props.updateAutocompleteSearchTerm
+              }
+              filters={filters}
+              toggleFilter={this.props.toggleFilter}
+              searchResults={searchResults}
+              eligibility={this.props.eligibility}
+              showModal={this.props.showModal}
+              eligibilityChange={this.props.eligibilityChange}
+            />
+          </div>
         ) : (
-          <InstitutionSearchForm
-            filtersClass={filtersClass}
-            search={search}
-            autocomplete={this.props.autocomplete}
-            location={this.props.location}
-            clearAutocompleteSuggestions={
-              this.props.clearAutocompleteSuggestions
-            }
-            fetchAutocompleteSuggestions={
-              this.props.fetchAutocompleteSuggestions
-            }
-            handleFilterChange={this.handleFilterChange}
-            updateAutocompleteSearchTerm={
-              this.props.updateAutocompleteSearchTerm
-            }
-            filters={filters}
-            toggleFilter={this.props.toggleFilter}
-            searchResults={searchResults}
-            eligibility={this.props.eligibility}
-            showModal={this.props.showModal}
-            eligibilityChange={this.props.eligibilityChange}
-          />
+          <div>
+            <div className="row">
```

Comment:
Same question here: Does it make sense to use a grid for this?

Reply:
old code, removing didn't affect display